lib/foreman_deployments/engine.rb

```diff
@@ -4,10 +4,7 @@
 
 module ForemanDeployments
   class Engine < ::Rails::Engine
-    config.autoload_paths += Dir["#{config.root}/app/controllers/foreman_deployments/concerns"]
-    config.autoload_paths += Dir["#{config.root}/app/helpers/foreman_deployments/concerns"]
-    config.autoload_paths += Dir["#{config.root}/app/models/foreman_deployments/concerns"]
-    config.autoload_paths += Dir["#{config.root}/app/overrides"]
+    config.autoload_paths += Dir["#{config.root}/app/**/"]
```

Comment:
The trick here is `Dir[]` expands the paths before they are assigned to the autoload paths.